h/static/styles/share-link.scss

```diff
@@ -12,7 +12,21 @@
 }
 
 .share-link:hover {
-  text-decoration: underline;
+  width: 100%;
+  color: $gray-dark;
+}
+
+.share-dialog-link {
```

Comment:
This is starting to get confusing now because we're mixing styles for three different components (the groups list, the share pane triggered by the drop-down and the in-card share dialog) in the same file. I'll merge this now but I suggest as a follow up extracting the share dialog out into its own component (so we'd have `share-dialog.scss`, `share_dialog.html` and `share-dialog.js`)